Dockerfile.rhel

```diff
@@ -0,0 +1,17 @@
+# base image source https://github.com/rhdt/EL-Dockerfiles/blob/master/base/python3/Dockerfile
+
+FROM prod.registry.devshift.net/osio-prod/base/python3:latest
+
+LABEL maintainer="Sarah Masud <smasud@redhat.com>"
+
+COPY ./requirements.txt /requirements.txt
+RUN pip3 install -r /requirements.txt && rm /requirements.txt
+
+COPY ./src /src
```

Comment:
I believe you will need to update this dockerfile once swagger PR is merged.

Reply:
Sure, I will update the Swagger PR as well.